### PR TITLE
Make 'BuildDocumentation' work recursively on folders.

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

**Comment by bobdenotter** [comment]
Before:

screen_shot_2016-01-31_at_13_30_27

After:

screen_shot_2016-01-31_at_13_50_39

File: src/Command/BuildDocumentation.php [hunk 1]
```diff
@@ -33,32 +33,55 @@ protected function configure()
...
 
+        foreach ($input->getArgument('branches') as $branch) {
+            $this->getSubTree($branch, $output, './source_docs/');
+            $versionsArray[$branch] = $foldername;
+        }
+
+        $dumper = new Dumper();
+
+        // Write the passed in versions to the yml file
+        $path = getcwd() . '/app/' . $this->versions;
```
Comment (GawainLynch) [inline src/Command/BuildDocumentation.php]: What if the binary is moved to another directory?

Comment (bobdenotter) [inline src/Command/BuildDocumentation.php]: The binary of what? This is just the docs site, so we can just choose not to do so, right?

Comment (GawainLynch) [inline src/Command/BuildDocumentation.php]: This is a `Command` class so it "creates" a "binary", or more explicitly something executable

Comment (GawainLynch) [inline src/Command/BuildDocumentation.php]: Also, technically it can, and should be able to, be called from anywhere

**Pull request merged**